receiver/prometheusremotewritereceiver/receiver_test.go

```diff
@@ -267,6 +267,80 @@ func TestTranslateV2(t *testing.T) {
 			}(),
 			expectedStats: remote.WriteResponseStats{},
 		},
+		{
+			name: "TODO",
```

Comment:
I don't know how to name it yet. Adding this to discuss what is better

Reply:
naming is hard xD

maybe `separate timeseries - same labels - should be same datapointslice`